resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DbReader.scala

```diff
@@ -60,6 +61,23 @@ object DbReader {
           )"""
       .query[Runtime]
 
+  val deletingRuntimeQuery =
+    sql"""SELECT DISTINCT c1.id, cloudContext, c1.cloudProvider, runtimeName, rt.cloudService, c1.status, rt.zone, rt.region
+          FROM CLUSTER AS c1
+          INNER JOIN RUNTIME_CONFIG AS rt ON c1.runtimeConfigId = rt.id
+          WHERE
+            c1.status = "Deleting" AND
+            c1.destroyedDate > now() - INTERVAL 60 MIN AND
```

Comment:
should this be `c1.dataAccessed < now() - INTERVAL 60 MIN`...basically making sure the runtime hasn't been accessed for more than 1 hr.

`destroyedDate` will always be the dummy tmestamp becuz these runtimes aren't deleted yet

Reply:
Good point, I will do that, thanks!